### PR TITLE
Update README.rst

### LINKED ISSUE #168: How could I enable the upload image in the ckeditor ?
I want to enable upload the image in the ckeditor, How could I set the the `CKEDITOR_SETTINGS`?

`TextEditorWidget` , about  `installed_plugins` parameter,  How should I to enable 'cms_plugins, djangocms_picture' plugins?

### PR DESCRIPTION
**Pull request by FinalAngel**
fixes #168, #160 and updates latest version to 3.2 requirements.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -3,31 +3,33 @@ djangocms-text-ckeditor
 
 Text Plugin for django-cms with CK-Editor.
 
-It currently supports:
+The latest Version of this package supports:
```
Comment (evildmp) [inline README.rst]: ``version``

File: README.rst [hunk 2]
```diff
@@ -3,31 +3,33 @@ djangocms-text-ckeditor
...
 
-* Django 1.6 - 1.8
-* django CMS 3.0 - 3.1
+* Django 1.8
```
Comment (yakky) [inline README.rst]: No more Django 1.6-1.7?

File: README.rst [hunk 3]
```diff
@@ -3,31 +3,33 @@ djangocms-text-ckeditor
...
+* django CMS 3.2+
 
 .. WARNING::
-   ``cms.plugins.text`` and ``djangocms-text-ckeditor`` can't be used at the same time.
+   For django CMS 3.0 and 3.1 use ``djangocms-text-ckeditor`` <= 2.7 (e.g.: version 2.7.0).
 
 .. WARNING::
-   For Django 1.4 and 1.5 use ``djangocms-text-ckeditor`` < 2.7.
```
Comment (yakky) [inline README.rst]: We need to keep this. 2.7 does not suppot Django 1.4 / 1.5 whatever the CMS version

File: README.rst (hunk 2 above)
Comment (FinalAngel) [inline README.rst]: nonononono +1 :dancer:

File: README.rst [hunk 4]
```diff
@@ -355,6 +357,11 @@ See the `html5lib documentation`_ for further information.
 .. _html5lib: https://pypi.python.org/pypi/html5lib
 .. _html5lib documentation: https://code.google.com/p/html5lib/wiki/UserDocumentation#Sanitizing_Tokenizer
 
+Search
+------
+
+djangocms-text-ckeditor works well along `aldryn-search <https://github.com/aldryn/aldryn-search>`_ to enable text content to be searchable using Haystack.
```
Comment (evildmp) [inline README.rst]: ``djangocms-text-ckeditor works well with `aldryn-search <https://github.com/aldryn/aldryn-search>`_ to make text content using Haystack``

**Pull request merged**